### PR TITLE
Prepare for v2.9

### PR DESCRIPTION
**Pull request by anjakefala**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
 # VisiData version history
 
+# 2.9 (2022-04-XX)
+
+- add XDG support (thanks @jck for the PR #1420)
+    - `options.config` default is now `"$XDG_CONFIG_HOME"/visidata/config.py` if `$XDG_CONFIG_HOME` is set and `config.py` exists. If not, falls back to the standard `/home/.visidatarc`.
```
Comment (saulpw) [inline CHANGELOG.md]: change `/home` to `$HOME`

File: visidata/__init__.py [hunk 2]
```diff
@@ -144,6 +145,10 @@ def getGlobals():
 import visidata.loaders.vds
 import visidata.loaders.odf
 import visidata.loaders.lsv
+import visidata.loaders.arrow
+import visidata.loaders.parquet
```
Comment (saulpw) [inline visidata/__init__.py:149]: also add `parquet` to formats list

File: visidata/_input.py [hunk 3]
```diff
@@ -42,7 +42,11 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, tb):
         with suppress(curses.error):
             curses.curs_set(0)
-            curses.mousemask(-1)
+            if options.mouse_interval:
```
Comment (saulpw) [inline visidata/_input.py:45]: this `-1` needs to be `0xffffffff`

File: visidata/deprecated.py [hunk 4]
```diff
@@ -138,6 +139,49 @@ def load_tsv(fn):
     vs = open_tsv(Path(fn))
     yield from vs.iterload()
 
+class VisiDataSheet(visidata.IndexSheet):
```
Comment (saulpw) [inline visidata/deprecated.py]: We need to get rid of this.

File: visidata/mainloop.py [hunk 5]
```diff
@@ -295,11 +325,11 @@ def initCurses():
     colors.setup()
 
     curses.noecho()
-    curses.cbreak()
 
     curses.raw()    # get control keys instead of signals
     curses.meta(1)  # allow "8-bit chars"
-    curses.mousemask(-1 if options.mouse_interval else 0)
+    curses.MOUSE_ALL = 0xffffffff
```
Comment (saulpw) [inline visidata/mainloop.py:331]: notice that we put this on the curses module for use later, so it's weird that it's not available in the other context handler.

**Review by saulpw — APPROVED** [review]
pending CHANGELOG

File: visidata/mainloop.py (hunk 5 above)
Comment (anjakefala) [reply visidata/mainloop.py:331]: test_commands.py does not go through `initCurses`!

Comment (saulpw) [reply visidata/mainloop.py:331]: Then let's just use `0xffffffff` as an inline constant in both places with the comment `# curses.MOUSE_ALL`.

**Comment by anjakefala** [comment]
We are going to move to a release branch!

**Pull request closed without merging**